english/udacity/udacity.md

```diff
@@ -1,3 +1,6 @@
 Alas, there is not, you're stuck. Get Caroline on it!
 
-Click [here](https://github.com/cbuckey-uda) to visit Caroline's github page!
\ No newline at end of file
+Click [here](https://github.com/cbuckey-uda) to visit Caroline's github page!
```

Comment:
This line points to a Github page outside of the create-your-own-adventure repo, and was already in the file that I edited.